src/work-allocation/release2/components/available-tasks-filter/available-tasks-filter.component.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { Location } from '@angular/common';
+import { Component, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
+import { of } from 'rxjs';
+
+import * as dtos from 'src/work-allocation/models/dtos';
```

Comment:
Should not import from `src/`.

Reply:
removed all /src imports

Reply:
Mostly, yes. Found one other and flagged it separately. :)